src/components/pages/WelcomePage.tsx

```diff
@@ -763,19 +765,33 @@ class WelcomePage extends React.Component<Props, State> {
   private setAutocompleteOptions = async (path: string, input: string) => {
 
     // Handle place autocomplete
-    if (path === "/linkedevents/places/search") {
-      if (input.length < 3) {
-        return [];
-      }
-      const res = await fetch(`https://mantyharju-test.linkedevents.fi/v1/search/?type=place&input=${ input }`);
-      const data = await res.json();
-      return data.data.map((place: any) => {
-        return {
-          name: place.name && place.name.fi ? place.name.fi : place.id,
-          value: place.id
-        };
-      });
+    if (input.length < 1) {
+      return [];
+    }
+
+    if (Object.keys(this.state.fetchData).length === 0) {
```

Comment:
Try using `if (!fetchData) ` because empty object `{}` should be falsey and get fetch data from state with `const { fetchData } = this.state;`

Reply:
Some reason behaviour is different